ddl/ddl_worker.go

```diff
@@ -618,6 +637,8 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 			}
 
 			w.setDDLLabelForTopSQL(job)
+			w.setDDLSourceForDiagnosis(job)
```

Comment:
What about combine it into `setDDLLabelForTopSQL` and change its name?

Reply:
`setDDLLabelForTopSQL` will attach task info to the current goroutine, which is not necessary for this PR's usage.